### ISSUE TITLE
Support alternative remote source repositories

### ISSUE THREAD

**Issue by r6p** [mguymon/naether#18]
Currently the maven repo url is hard-coded in the code below. This should be configurable in-case you want to provide an alternative like an "artifactory" instance. It also seems odd to download the jars at install time "bundle install" instead of just packaging the jars in the gem.

bootstrap.rb
````
              maven_mirror = "http://repo1.maven.org/maven2...
````

**Comment by mguymon** [comment]
So the reason is two fold. There are 29 jar dependencies for Naether, coming in at 3 megs. If the jars are packaged as part of the gem, they would get downloaded again and again with every single major, meager, and minor gem change. If they are stored in the Maven Repo, they are only downloaded once and if updated, only the updated jar is downloaded. In addition, it is _Eating your own dog food_. Naether is a runtime for resolving and managing jar dependencies, it should take advantage of the services it provides.

As to the main maven repo being the only source, maybe allow an ENV override? Something along the lines of 

    maven_mirror = ENV['NAETHER_MIRROR'] || "http://repo1.maven.org/maven2"

then to install the gem, you would have to

NAETHER_MIRROR=http://my.local.repo gem install naether

**Comment by r6p** [comment]
Ok good points above. I like your suggestion on an environment variable as long as it will work when using bundler too, as in:

````
NAETHER_MIRROR=http://my.local.repo bundle install
````

**Comment by mguymon** [comment]
released as 0.14.0
